enhancements/monitoring/hypershift-monitoring.md

```diff
@@ -0,0 +1,384 @@
+
+For example:
+
+* For our Managed HSRE, this can be our RHOBS service.
+* For HSREs other than SD members, it can be locally deployed [Observatorium](http://observatorium.io/) or 3rd-party vendor like Grafana Cloud, Amazon or Google Managed Prometheus, Logz.io and others.
+
+A high-level view would look like this:
+
+![h](assets/hypershift-highlvl.png)
+
+The model as above allows:
+
+* Metrics Centralization
+* Extensibility (e.g. one can choose different Observability Products or switch them on demand).
+* Separation of concerns which should result in:
+  * Higher velocity of the Observability Product
+  * Higher velocity of OSD as the offering has to care less about in-cluster observability flows.
+  * Allows managing two products by separate teams without conflicts if one chooses to.
+
+> Note that this does not imply "Centralized Observability Product" to run in a separate cluster and be managed by a separate team. We can mix and match as needed, but strictly for Managed HyperShift use cases we would go for RHOBS service fully managed by Observability Group with AppSRE (SD).
+
+Reasons for choosing RHOBS for our HSRE:
+
+* We can have tighter control on observability budget and spending
+* It’s our common, Red Hat responsibility to offer managed OpenShift. If Observability Group + AppSRE will not provide their support and time, the same (or more) effort and money has to be spent on OSD side (or vendor)
+* The monitoring Team already manages this service.
+  * We already work closely together with AppSRE, tightly integrated with them and familiar with AppInterface.
+  * We have devs on-call and SRE mindset.
+
+## Design Details
+
+In details the full design looks as follows:
+
+![h](assets/hypershift-details.png)
+
+Let’s take a look at all parts:
+
+1. Hosted control planes does not deploy any own monitoring, other than relevant Service Monitors to be scraped.
+2. For all control planes we propose to deploy [Monitoring Stack Operator](https://github.com/rhobs/monitoring-stack-operator) that will deploy set of Prometheus/Prometheus Agents that will be sending data off cluster to RHOBS. No local alerting and local querying will be allowed. This path will forward both monitoring data as well as part of telemetry relevant for Telemeter.
+3. On data-plane we propose running roughly unchanged CMO stack with platform monitoring and optional UWM. This allows feature parity for customers as well as ability to remote write few metrics interersting for HSRE use (as specified in Goal). The unique part is that Platform Monitoring will know nothing about master nodes and control plane resources (etcd, control plane operators, Kube services etc). It will only provide "platform" metrics for worker nodes and containers running there. Part of telemetry related to data-plane will be sent through Telemeter client as usual. We assume CMO can be deployed on Management cluster for easier maintenance.
```

Comment:
CMO for the data plane is currently deployed on the worker nodes. Are you proposing that we move it to the control plane? What do we gain?

Reply:
> We assume CMO can be deployed on the Management cluster for easier maintenance.

"Easier maintenance" was the sentence. We don't see any other strong motivation in either direction, so up to you.